File: app/src/main/java/com/example/unifi/signUP_Cat.java
package com.example.unifi;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class signUP_Cat extends AppCompatActivity {
    private Button aspring,existing;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_sign_up_cat);
        aspring = findViewById(R.id.aspring);
        existing = findViewById(R.id.existing);

        aspring.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent i = new Intent(signUP_Cat.this , signup.class);
                startActivity(i);
            }
        });
        existing.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent i = new Intent(signUP_Cat.this , signup_existing.class);
                startActivity(i);
            }
        });
    }
}

File: app/src/main/java/com/example/uni_fikotlin/Auth/Signup_cat.kt
package com.example.uni_fikotlin.Auth

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import com.example.uni_fikotlin.R

class signup_cat : AppCompatActivity() {
    private lateinit var signup_aspring: Button
    private lateinit var signupExisting: Button
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_signup_cat)
        signup_aspring = findViewById(R.id.aspring)
        signupExisting = findViewById(R.id.existing)

        signup_aspring.setOnClickListener {

            startActivity(Intent(this, com.example.uni_fikotlin.Auth.signup_aspring::class.java))
        }
    }
}